js/views/stock.js

```diff
@@ -1 +1,19 @@
+$(document).ready(function(){
 // define your StockView here
+
+var StockView = function(model){
+  this.model = model
+  this.render()
+}
+
+StockView.prototype = {
+  render: function(){
+    // this.model.price
+    // this.model.companyName etc...
+    // nofrickingclueammiegetyershittogether
```

Comment:
yes! Your pseudocode looks great! Please let me know if you'd like to work through this solution together.